src/components/TeamGeneratorPage/PlayerCard.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 
-const playerCard = ({ player, onRemovePlayer }) => {
+const playerCard = ({ player, onRemovePlayer, team }) => {
 	const drag = (e) => {
```

Comment:
this is the eventListener?

Reply:
`drag` is a callback function for the `onDragStart` event